scripts/build_utils.sh

```diff
@@ -960,7 +960,7 @@ build_debs() {
 
     echo building debs for $DIST
 
-    CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS $(extra_cmake_args)"
+    CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS $(extra_cmake_args)" + " -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON"
```

Comment:
IIUC, `scripts/build_utils.sh` is used by the CI for building packages. and developers always just use the packages for testing and to redistributing Ceph. if that's the case, who will be reading the output of the linter?

Reply:
#2269,. as mentioned above, is meant to be a check job, but if it's got to run after a ceph build, I'm not sure how it fits in either.